invoke.mysql.yml

```
@@ -1,7 +1,7 @@
 ---
 nautobot_capacity_metrics:
   project_name: "nautobot-capacity-metrics"
-  nautobot_ver: "2.4.20"
+  nautobot_ver: "next"  # TODO: Change to 3.0.0 after v3.0.0 is released
```

This should be changed to 3.0.0 now

```suggestion
  nautobot_ver: "3.0.0"
```